sessions/recap-project-2/example-implementation/form.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="form.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Add Question</h1>
+    </header>
+    <main>
+      <form class="question-form" data-js="form">
+        <label class="question-form__input">
+          <span>Question</span>
```

Comment:
Does the `span` have a purpose here? 🤔 

```suggestion
         Question
```